platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceImpl.java

```diff
@@ -100,13 +100,18 @@
    *     about bundle status for deployment services (like blueprint and spring).
    */
   public ApplicationServiceImpl(List<BundleStateService> bundleStateServices) {
+    this.bundleStateServices = bundleStateServices;
+    ignoredApplicationNames = new HashSet<>();
+
     BundleContext context = getContext();
+    if (context == null) {
+      LOGGER.warn("There is no context for this service");
+      return;
```

Comment:
The object has not been initialized properly at this point. I Think we might want to throw an IllegalStateException here.